pom.xml

```diff
@@ -231,6 +231,23 @@ file comparators, endian transformation classes, and much more.
       <version>4.12</version>
       <scope>test</scope>
     </dependency>
+    <!-- annotations from the Checker Framework: nullness, interning, locking, ... -->
+    <dependency>
+      <groupId>org.checkerframework</groupId>
+      <artifactId>checker-qual</artifactId>
+      <version>2.1.11</version>
```

Comment:
Update this version to 2.1.14 (though actually the following version is needed, because the following version adds JDK annotations for MessageFormat).